tests/test_oauth.py

```diff
@@ -233,8 +233,8 @@ def test_get_authorize_url(self):
         Testing get_authorize_url()
         """
         expected = 'https://accounts.spotify.com/authorize?client_id=thisisarealclientid&' \
-                   'response_type=code&redirect_uri=https%3A%2F%2Fthis-is-a-real-redirect.uri&' \
-                   'scope=user-modify-playback-state+ugc-image-upload+user-library-modify'
+                   'response_type=code&redirect_uri=https%3A%2F%2Fthis-is-a-real-redirect' \
+                   '.uri%3A0&scope=user-modify-playback-state+ugc-image-upload+user-library-modify'
```

Comment:
Good idea to convert to triplets. Is the rest of the codebase converted? An issue could maybe be created for this.

Reply:
They are changed where relevant. For API requests, the `requests` module handles this.

Reply:
Okay, cool